### PR TITLE
Added github actions for edge test for cloud KMS systems

### PR DESCRIPTION
**Pull request by shtripat**
This would trigger while PR merge to master and at the moment would execute edge tests for AWS and Azure.

### CODE REVIEW

File: .github/workflows/edge-test.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        check-latest: true
+      id: go
+    - name: Check out code
+      uses: actions/checkout@v3
+    - name: Test
+      env:
+        GO111MODULE: on
+      run: |
+         cd ./edge
+         go test -v -aws.config=/etc/kes/config-aws.yml -run="TestAWS"
```
Comment (aead) [inline .github/workflows/edge-test.yml:26]:
```
go test ./edge -v -aws.config=/etc/kes/config-aws.yml -run="TestAWS"
```

File: .github/workflows/edge-test.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        check-latest: true
+      id: go
+    - name: Check out code
+      uses: actions/checkout@v3
+    - name: Test
+      env:
+        GO111MODULE: on
+      run: |
+         cd ./edge
+         go test -v -azure.config=/etc/kes/config-azure.yml -run="TestAzure"
```
[the remark this answers is not in the document]
Comment (shtripat) [reply .github/workflows/edge-test.yml:44]: Ack

File: .github/workflows/edge-test.yml (hunk 1 above)
Comment (shtripat) [reply .github/workflows/edge-test.yml:26]: Ack

File: .github/workflows/edge-test.yml [hunk 3]
```diff
@@ -0,0 +1,44 @@
+name: edge tests
```
Comment (aead) [inline .github/workflows/edge-test.yml:1]: `Edge`

**Review by aead — APPROVED** [review]

**Pull request merged**